src/pollution_model/run_medusa_2.py

```diff
@@ -0,0 +1,525 @@
+
+    Parameters
+    ----------
+    tss_surface: float
+        total suspended solids of this surface
+
+    Returns
+    -------
+    Tuple[float, float]
+       Returns the total copper and zinc loads for this surface
+       [Total Copper, Total Zinc]
+    """
+    # Define constants
+    proportionality_constant_cu = 0.441
+    proportionality_constant_zn = 1.96
+    total_cu_load = tss_surface * proportionality_constant_cu
+    total_zn_load = tss_surface * proportionality_constant_zn
+    # Return total copper load, total zinc load
+    return total_cu_load, total_zn_load
+
+
+def dissolved_metal_load(total_copper_load: float, total_zinc_load: float,
+                         surface_type: SurfaceType) -> Tuple[float, float]:
+    """
+    Calculate the dissolved metal load for all surfaces from their total suspended solids.
+
+    Parameters
+    ----------
+    total_copper_load: float
+        total copper load for the surface
+    total_zinc_load: float
+        total zinc load for the surface
+    surface_type: int
+        The type of surface that we are calculating this for
+
+    Returns
+    -------
+    Tuple[float, float]
+        Returns the dissolved copper and zinc load for this surface
+        [Dissolved Copper Load, Dissolved Zinc Load]
+
+    Raises
+    ----------
+    ValueError
+        If the surface type is not a roof or road (concrete, copper, galvanised roof, asphalt road, or car park).
+    """
+    # Define error message (if needed)
+    invalid_surface_error = (f"Given surface is not valid for computing dissolved metal load."
+                             f" Needed a roof or road, but got {SurfaceType(surface_type).name}.")
+    # Set constant values based on surface type
+    match surface_type:
+        case SurfaceType.CONCRETE_ROOF:
+            f = 0.46
+            g = 0.67
+        case SurfaceType.COPPER_ROOF:
+            f = 0.77
+            g = 0.72
+        case SurfaceType.GALVANISED_ROOF:
+            f = 0.28
+            g = 0.43
+        case SurfaceType.ASPHALT_ROAD | SurfaceType.CAR_PARK:
+            f = 0.28
+            g = 0.43
+        case _:
+            raise ValueError(invalid_surface_error)
+    return f * total_copper_load, g * total_zinc_load
+
+
+def get_building_information(engine: Engine, area_of_interest: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
+    """
+    Extract relevant information about buildings from the database.
+    Then formats them such that they are easy to use for pollution modeling purposes.
+
+    Parameters
+    ----------
+    engine: Engine
+        The sqlalchemy database connection engine
+    area_of_interest : gpd.GeoDataFrame
+        A GeoDataFrame polygon specifying the area of interest to retrieve buildings in.
+
+    Returns
+    -------
+    gpd.GeoDataFrame
+        A GeoDataFrame containing rows corresponding to buildings, and columns corresponding to
+        attributes (Index, SurfaceArea, SurfaceType)
+    """
```

Comment:
I saw the linting errors and this will resolve them
```suggestion
def get_building_information(_engine: Engine, _area_of_interest: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
    """
    Extract relevant information about buildings from central_buildings.geojson, since the input data is not finalised.
    Then formats them such that they are easy to use for pollution modeling purposes.
    
    Github Issue to resolve the input_data: https://github.com/GeospatialResearch/Digital-Twins/issues/198

    Returns
    -------
    gpd.GeoDataFrame
        A GeoDataFrame containing rows corresponding to buildings, and columns corresponding to
        attributes (Index, SurfaceArea, SurfaceType)
    """
```